projects/plugins/jetpack/modules/copy-post.php

```diff
@@ -317,10 +317,12 @@ public function add_row_action( $actions, $post ) {
 		);
 		$edit_action = array(
 			'jetpack-copy' => sprintf(
-				'<a href="%s" aria-label="%s">%s</a>',
+				'<a href="%s" aria-label="%s">%s <img src="%s" alt="%s" width="16" height="16" style="vertical-align:text-bottom"></a>',
```

Comment:
Do you think we could switch to numbered placeholders? It's starting to be quite a few placeholders, and I think numbers may make things easier to quickly parse.